files/default/configure_nw_interface.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
```

Comment:
I'd add an initial comment describing what is the scope of this script.

Reply:
Done